pkg/mdformatter/linktransformer/link.go

```diff
@@ -207,32 +207,48 @@ func NewValidator(ctx context.Context, logger log.Logger, linksValidateConfig []
 		}
 	}
 
+	linktransformerMetrics := newLinktransformerMetrics(reg)
+	collector := colly.NewCollector(colly.Async(), colly.StdlibContext(ctx), colly.MaxDepth(1))
```

Comment:
So setting `colly.MaxDepth` as 1 here has no effect for mdox. colly has the ability to scrape links from websites recursively, but mdox only uses the colly Collector.Visit [method](https://github.com/bwplotka/mdox/blob/main/pkg/mdformatter/linktransformer/validator.go#L55), which by [default always has a depth of 1](https://github.com/gocolly/colly/blob/master/colly.go#L493).